sql-plugin/src/main/31xdb/scala/com/nvidia/spark/rapids/shims/v2/AQEUtils.scala

```diff
@@ -18,13 +18,11 @@ package com.nvidia.spark.rapids.shims.v2
 
 import org.apache.spark.sql.catalyst.expressions.Attribute
 import org.apache.spark.sql.execution.adaptive.{QueryStageExec, ShuffleQueryStageExec}
-import org.apache.spark.sql.execution.exchange.ReusedExchangeExec
 
 /** Utility methods for manipulating Catalyst classes involved in Adaptive Query Execution */
 object AQEUtils {
   /** Return a new QueryStageExec reuse instance with updated output attributes */
   def newReuseInstance(sqse: ShuffleQueryStageExec, newOutput: Seq[Attribute]): QueryStageExec = {
-    val reusedExchange = ReusedExchangeExec(newOutput, sqse.shuffle)
-    ShuffleQueryStageExec(sqse.id, reusedExchange, sqse.originalPlan)
```

Comment:
this change seems out of place and not related to shim work?

Reply:
Yes, this change is to make the DB build pass. I can spin this out to it's own PR.

Reply:
the DB build fails without these v2 changes?

Reply:
Yes

Reply:
then yes it should be a separate PR and should have details why.. there isn't an open issue for this?  I haven't noticed the DB builds failing but builds have been messed up lately.